pkg/util/ipvs/ipvs_test.go

```diff
@@ -380,3 +384,35 @@ func TestFrontendDestinationString(t *testing.T) {
 		}
 	}
 }
+
+func TestGetRequiredIPVSModules(t *testing.T) {
+	Tests := []struct {
+		name          string
+		kernelVersion *version.Version
+		want          []string
+	}{
+		{
+			name:          "kernel version < 4.19",
+			kernelVersion: version.MustParseGeneric("4.18"),
+			want:          []string{KernelModuleIPVS, KernelModuleIPVSRR, KernelModuleIPVSWRR, KernelModuleIPVSSH, KernelModuleNfConntrackIPV4},
+		},
+		{
+			name:          "kernel version 4.19",
+			kernelVersion: version.MustParseGeneric("4.19"),
+			want:          []string{KernelModuleIPVS, KernelModuleIPVSRR, KernelModuleIPVSWRR, KernelModuleIPVSSH, KernelModuleNfConntrack},
+		},
+	}
+	for _, test := range Tests {
+		t.Run(test.name, func(t *testing.T) {
+			got := GetRequiredIPVSModules(test.kernelVersion)
+			if len(got) != len(test.want) {
```

Comment:
`reflect.DeepEqual` check below should cover this case

Reply:
It left over from previous version, i will fix it.